.github/workflows/frontend.yml

```diff
@@ -0,0 +1,31 @@
+name: Check search frontend
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - '**'
+
+jobs:
+  check-search-frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bring system up
+        run: |
```

Comment:
Do we also need to make sure npm is installed prior to running docker-compose up?

Reply:
What doesn't make sense to me is that npm is run in the container, so what's available on the host machine is irrelevant.